packages/async-rewriter/src/async-writer-babel.ts

```diff
@@ -257,6 +274,15 @@ var TypeInferenceVisitor: Visitor = { /* eslint no-var:0 */
             const tyTS = lhsNode as unknown;
             const id = tyTS as babel.types.Identifier;
             state.symbols.updateAttribute(id.name, attrs, sType);
+          } else if (lhsNode.type === 'ThisExpression') {
+            const classPath = path.findParent((p) => p.isClassDeclaration());
+            if (!classPath) {
+              throw new MongoshUnimplementedError('Unable to handle this outside of method definition of class declaration');
+            }
+            if (attrs.length > 1) {
+              throw new MongoshUnimplementedError('Unable to handle multi-layered assignment to \'this\'');
```

Comment:
^ yea something like your above statement for all the mentions of 'this' in errors would be really good.

Reply:
Cool, I replaced each `this` with `'this' keyword` in the error messages